src/libbootimg.c

```diff
@@ -171,31 +182,88 @@ void libbootimg_destroy(struct bootimg *b)
     }
 }
 
+uint32_t libbootimg_elf_arrtoval(uint8_t* tab)
+{
+    return (tab[3]<<24) + (tab[2]<<16) + (tab[1]<<8) + (tab[0]);
```

Comment:
Absolutely, and I had that working... I don't know where it went, will look for it or redo, then update.